tokio/src/runtime/builder.rs

```diff
@@ -93,6 +93,8 @@ pub struct Builder {
     /// How many ticks before yielding to the driver for timer and I/O events?
     pub(super) event_interval: u32,
 
+    pub(super) local_queue_capacity: usize,
```

Comment:
This PR makes the local queue capacity configurable but does not expose the option. At some point, it might, but right now, it is for tests.